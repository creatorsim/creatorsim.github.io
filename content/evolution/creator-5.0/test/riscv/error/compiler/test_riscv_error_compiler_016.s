
#
# Creator (https://creatorsim.github.io/creator/)
#

 .data
 	.align 2
    example:	.string hello world"

