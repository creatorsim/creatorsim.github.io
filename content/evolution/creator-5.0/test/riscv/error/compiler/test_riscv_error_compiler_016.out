

Error: Unterminated string literal
   ╭─[ assembly:8:34 ]
   │
 8 │     example:    .string hello world"
   │                                     ┬  
   │                                     ╰── Caused by this
   │                                     │  
   │                                     ╰── while parsing this token
───╯

Not executed
keyboard[0x0]:''; display[0x0]:''; 

